dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/deletefiles/DeleteItemPresenter.java

```diff
@@ -166,18 +166,18 @@ private void constructMessageForLog() {
         String alertMessage = null;
 
         if (deleteTask instanceof Ds3DeleteBucketTask) {
-            alertMessage = resourceBundle.getString("deleteBucketErr");
+            alertMessage = "deleteBucketErr";
         } else if (deleteTask instanceof Ds3DeleteFoldersTask) {
-            alertMessage = resourceBundle.getString("folderDeleteFailed");
+            alertMessage = "folderDeleteFailed";
         } else if (deleteTask instanceof Ds3DeleteFilesTask) {
-            alertMessage = resourceBundle.getString("deleteFailedError");
+            alertMessage = "deleteFailedError";
         }
         message = alertMessage + StringConstants.SPACE + deleteTask.getErrorMsg();
```

Comment:
if alertMessage is still null, won't this throw a null pointer exception?

Reply:
This is a good catch.  At a minimum it should have an else condition, or a default value.  That would prevent the potential for a npe.